firmware/base2015/PCLink.hpp

```diff
@@ -0,0 +1,24 @@
+#include <mbed.h>
+#include <USBHID.h>
+
+class PCLink {
+public:
+    static const uint16_t RJ_VENDOR_ID = 0x524A;
+    static const uint16_t RJ_PRODUCT_ID = 0x4253;
+    static const uint16_t RJ_RELEASE = 0x0000;
+
+    PCLink(void);
+    PCLink(uint16_t vendorID, uint16_t productID, uint16_t release);
+    virtual ~PCLink(void);
+    void setSerialDebugging(Serial *pc);
+    void setLed(DigitalOut *led);
+    void read(void);
+    void reply(void);
+private:
+    Serial *pc;
```

Comment:
Let's name instance variables with a leading underscore to keep consistency with the rest of our c++.